tests/js/server/replication2/replication2-maintenance-replicated-state-cluster.js

```diff
@@ -210,12 +191,61 @@ const replicatedStateSuite = function (stateType) {
   };
 };
 
-const suiteWithState = function (stateType) {
+const replicatedStateDropSuite = function (stateType) {
+
+  const getLocalStatus = function (serverId, database, logId) {
+    let url = LH.getServerUrl(serverId);
+    const res = request.get(`${url}/_db/${database}/_api/replicated-state/${logId}/local-status`);
+    return res.json;
+  };
+
+  return {
+    setUpAll, tearDownAll, setUp, tearDown,
+
+    ["testDropLogOnly_" + stateType]: function () {
+      const logId = LH.nextUniqueLogId();
+      const servers = _.sampleSize(LH.dbservers, 3);
+      const leader = servers[0];
+      createReplicatedState(database, logId, servers, leader, stateType);
+      LH.waitFor(spreds.replicatedStateIsReady(database, logId, servers));
+
+      LH.waitFor(function () {
+        for (const server of servers) {
+          const response = getLocalStatus(server, database, logId);
+          if (response.code !== 200) {
+            return Error(`Server ${server} returned code ${response.code}, expected ${200}`);
+          }
+        }
+        return true;
+      });
+
+      LH.replicatedLogDeletePlan(database, logId);
```

Comment:
I find name of this test is a bit confusing. Did you intend to drop the replicated log in here? I think this line should not have any effect on the outcome of the test.